ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -480,3 +480,38 @@ and duration >= 180000
     ]
   }
 }
+
+resource "azurerm_monitor_scheduled_query_rules_alert" "frontend_fail_to_communicate_with_backend_alert" {
+  name                = "${var.env}_frontend_fail_to_communicate_with_backend_alert"
+  description         = "Action will be triggered when API request to feature flag endpoint failures > 0"
+  location            = data.azurerm_resource_group.app.location
+  resource_group_name = var.rg_name
+  severity            = var.severity
+  frequency           = 60
```

Comment:
set this to 60 minutes so that we evaluate this rule every hour so we are not inundated with alerts
[see frequency](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/monitor_scheduled_query_rules_alert#frequency)